options/locale/locale_en-US.ini

```diff
@@ -1583,8 +1583,14 @@ teams.write_permission_desc = This team grants <strong>Write</strong> access: me
 teams.admin_permission_desc = This team grants <strong>Admin</strong> access: members can read from, push to and add collaborators to team repositories.
 teams.repositories = Team Repositories
 teams.search_repo_placeholder = Search repository…
-teams.add_team_repository = Add Team Repository
+teams.add_team_repository = Add
 teams.remove_repo = Remove
+teams.add_all_repos = Add All
+teams.remove_all_repos = Remove All
```

Comment:
Can't we reuse general strings "add", "remove all", ...

Reply:
I've moved them to general strings.